webapp/src/game/game.service.ts

```diff
@@ -76,4 +86,20 @@ export class GameService {
     const gameState = { ...game.gameState, isRunning: false };
     return this.gameRepository.update(id, { ...game, gameState });
   }
+
+  async setWhitePlayer(id: number, player: User) {
+    const game = await this.findOne(id);
+    return this.gameRepository.update(id, { ...game, player_white: player });
+  }
+
+  async setBlackPlayer(id: number, player: User) {
+    const game = await this.findOne(id);
+    return this.gameRepository.update(id, { ...game, player_black: player });
+  }
+
+  isGameBelongToPlayer(game: Game, player: User) {
```

Comment:
Weird english here. Besides, it's a function, it **does** things so prefer a verb: `checkGameBelongToPlayer`?